opencensus/pom.xml

```diff
@@ -39,6 +39,20 @@
     <opencensus.version>0.31.1</opencensus.version>
   </properties>
 
+ <!--  Using libraries-bom to manage versions.
+  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>25.0.0</version>
```

Comment:
Please update to the latest version: https://search.maven.org/artifact/com.google.cloud/libraries-bom/26.1.3/pom

Reply:
Done!